napari_ome_zarr/_tests/test_reader.py

```diff
@@ -50,7 +61,7 @@ def test_get_reader_pass(self):
     def assert_layers(self, layers, visible_1, visible_2, path="path_3d"):
         # TODO: check name
 
-        assert len(layers) == 2
+        assert len(layers) == self.n_datasets
```

Comment:
```suggestion
        assert len(layers) == self.n_layers
```
@dstansby Should this be `n_layers`? it's that or the initdir needs `self.n_datasets` to be defined.

Reply:
Woops, I think this was just a typo